spinalcordtoolbox/scripts/sct_get_centerline.py

```diff
@@ -214,11 +215,21 @@ def main(argv: Sequence[str]):
         return
 
     # Extrapolate and regularize (or detect if optic) cord centerline
-    im_centerline, arr_centerline, _, _ = get_centerline(im_labels,
-                                                         param=param_centerline,
-                                                         space=arguments.space,
-                                                         verbose=verbose,
-                                                         remove_temp_files=arguments.r)
+    im_centerline, _, _, _ = get_centerline(im_labels,
+                                            param=param_centerline,
+                                            # NB: We intentionally skip the pix->phys conversion here, since we want to
+                                            #     convert to phys AFTER the image has been reoriented away from RPI.
+                                            # space=arguments.space,
+                                            verbose=verbose,
+                                            remove_temp_files=arguments.r)
+
+    # Fetch `arr_centerline` from `im_centerline`
+    # (This is the easiest way to get coords that are in their original orientation. This is largely equivalent to how
+    #  arr_centerline is determined inside `get_centerline`, but without the RPI orientation.)
+    arr_centerline = np.array(find_and_sort_coord(im_centerline))
+    # Convert from pixel coordinates to physical coordinates (in original orientation)
+    if arguments.space is "phys":
```

Comment:
<img width="1423" alt="Screenshot 2024-09-04 at 8 57 59 PM" src="https://github.com/user-attachments/assets/b52332ae-9dc2-46dd-ada3-95d62e7bb0a6">


```suggestion
    if arguments.space == "phys":
```

Reply:
The output coordinates in the ".csv" are in pixels even though I have `-space phys`. This is probably why.